Gemfile

```diff
@@ -23,7 +23,8 @@ gem 'puma', '~> 5.0'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'jsonapi.rb', '~> 1.7'
+gem 'pry', '~> 0.14.1'
```

Comment:
Wouldn't it be better to add it to `group :development, :test`?

Reply:
> Wouldn't it be better to add `pry` to `group :development, :test`?

@rafalrothenberger done :)